src/openvic-simulation/InstanceManager.cpp

```diff
@@ -39,10 +39,16 @@ void InstanceManager::update_gamestate() {
 
 	Logger::info("Update: ", today);
 
+	map_instance.update_modifier_sums(today, definition_manager.get_modifier_manager().get_static_modifier_cache());
```

Comment:
Get the manager once and get the static modifier cache once. You can store them in a local variable and reuse it.

Reply:
Yeah it might be worth switching over to that approach for arguments that are always the same, however the current approach is used all over the codebase so it's better to stay consistent here and make the change everywhere in one go in a separate PR if we decide to go that way.